modules/insights-operator-about.adoc

```diff
@@ -8,12 +8,12 @@
 
 The Insights Operator periodically gathers configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry.
 
-Users of {product-title} can display the report of each cluster in {cluster-manager-url}. If any issues have been identified, Insights provides further details and, if available, steps on how to solve a problem.
+Users of {product-title} can display the report of each cluster in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]. If any issues have been identified, Insights provides further details and, if available, steps on how to solve a problem.
```

Comment:
In many other areas of the OCP documentation, https://console.redhat.com/openshift is referred to as OpenShift Cluster Manager. The "{cluster-manager-url}" attribute is typically used to create linked text that uses that name.

You could consider replacing "link:[https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]" with "{cluster-manager-url}" for consistency with this docs library.

Reply:
created attributes and updated in all locations with:` the {insights-advisor-url} service on {hybrid-console}.`